src/opencensus.erl

```diff
@@ -185,8 +185,7 @@ put_attributes(NewAttributes, Span=#span{attributes=Attributes}) ->
 %%--------------------------------------------------------------------
 -spec generate_trace_id() -> non_neg_integer().
 generate_trace_id() ->
-    <<Id:128>> = crypto:strong_rand_bytes(16),
-    Id.
+    rand:uniform(340282366920938463463374607431768211456).
```

Comment:
This will lead to icky results on erlang pre 20, I think we'd need a *ghasp* `-ifdef` here! Fortunately, there is https://github.com/project-fifo/rebar_erl_vsn that makes this easy (end of shameless plug).